plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/BaseKuduDistributedQueries.java

```diff
@@ -15,38 +15,19 @@
 
 import io.trino.testing.AbstractTestDistributedQueries;
 import io.trino.testing.MaterializedResult;
-import io.trino.testing.QueryRunner;
 import io.trino.testing.sql.TestTable;
 import org.testng.SkipException;
-import org.testng.annotations.AfterClass;
 import org.testng.annotations.Test;
 
 import java.util.Optional;
 
-import static io.trino.plugin.kudu.KuduQueryRunnerFactory.createKuduQueryRunnerTpch;
 import static io.trino.spi.type.VarcharType.VARCHAR;
 import static io.trino.testing.MaterializedResult.resultBuilder;
 import static io.trino.testing.assertions.Assert.assertEquals;
 
-public class TestKuduDistributedQueries
+public abstract class BaseKuduDistributedQueries
```

Comment:
Can we instead use base `BaseKuduDistributedQueries ` no `BaseConnectorTest`?

Reply:
Added an additional commit to extend `BaseConnectorTest`.